packages/sdk/contractkit/src/wrappers/Attestations.ts

```diff
@@ -599,7 +581,16 @@ export class AttestationsWrapper extends BaseWrapper<Attestations> {
    * @param serviceURL: validator's attestation service URL
    * @param body
    */
-  revealPhoneNumberToIssuer(serviceURL: string, requestBody: AttestationRequest) {
+  async revealPhoneNumberToIssuer(serviceURL: string, requestBody: AttestationRequest) {
+    const statusResponse = await fetch(appendPath(serviceURL, 'status'))
+    const statusResponseBody = await statusResponse.json()
+    const version = statusResponseBody.version
+
+    // remove signature field if validator's attestation service is version 1.5.0 or lower
+    if (version.localeCompare('1.5.0', undefined, { numeric: true, sensitivity: 'base' }) <= 0) {
+      delete requestBody.phoneNumberSignature
```

Comment:
I don't think this is necessary. Deserialization should complain if a required field is missing, but it should safely ignore extra unexpected fields. To test it, I threw together this code snippet:
https://playcode.io/890789

Reply:
Thanks @codyborn ! That code snippet looks right to me. @isabellewei what were the errors you were seeing when this extra field was provided?

Reply:
Eela helped me test it with a locally running AS version 1.5.0 and sending a request with an extra parameter does indeed work with no errors, thanks @codyborn :)